<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PipUI</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_startup">
    <widget class="QPushButton" name="pushButton_discoverpackages">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon-images/discoverpackages-wingetUI200p.png</normaloff>icon-images/discoverpackages-wingetUI200p.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_installedpackages">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon-images/installedpackages-wingetUI200p.png</normaloff>icon-images/installedpackages-wingetUI200p.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_softwareupdates">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>71</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icon-images/softwareupdates-wingetUI200p.png</normaloff>icon-images/softwareupdates-wingetUI200p.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_python2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Python2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_python3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Python3</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_discoverpackages">
    <widget class="QPushButton" name="pushButton_back2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pypi">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Take Me to PyPi.org</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_installedpackages">
    <widget class="QListWidget" name="listWidget_installedpippackages">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_installedpippackages">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Installed Pip Packages</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_back1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_softwareupdates">
    <widget class="QPushButton" name="pushButton_back3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_updatepippackages">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_updatepippackages">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>Update pip packages</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
